src/presets/cloudflare.ts

```diff
@@ -19,6 +20,7 @@ const cloudflareNodeCompatModules = [
   "stream/promises",
   "stream/web",
   "string_decoder",
+  "sys",
```

Comment:
can we revert this together with l 50-52?

- the PR is "update buffer to be cloudflare compat module", let's not mixed things
- if `sys` is an alias of `util`, why do we want `util` to be a node compat module while `sys` is hybrid